resources/doxygen_extra.css

```diff
@@ -0,0 +1,101 @@
+/* s3://drake-web/fonts/stylesheet.css */
+@import url('https://dmn3132s4wr9m.cloudfront.net/fonts/stylesheet.css');
```

Comment:
Can we make a copy an add the proper license? I would prefer to be as autonomous as possible with this.

Reply:
So, that's trickier than it seems. It's not just a stylesheet, but all the fonts too. I guess I can try replicate them all and rewrite SASS to hit those instead.

Reply:
Alternatively, we can drop these custom fonts :)

Reply:
+1 to drop.

Reply:
Dropping the entire stylesheet (it was only used to add these fonts) in https://github.com/ToyotaResearchInstitute/ament_cmake_doxygen/pull/1/commits/5ce4b79b6ccae7c8be08097f817f22d585f1e0bf